<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="displayButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>429</y>
      <width>959</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Display list</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startRealTimeButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>959</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start real-time</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>951</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="nameButton">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PIDButton">
       <property name="text">
        <string>PID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="parentPIDButton">
       <property name="text">
        <string>Parent PID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="privateUsageButton">
       <property name="text">
        <string>Private Usage</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="threadsButton">
       <property name="text">
        <string>Threads</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="priorityButton">
       <property name="text">
        <string>Priority Base</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="priorityClassButton">
       <property name="text">
        <string>Priority class</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>951</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTextBrowser" name="nameBrowser"/>
     </item>
     <item>
      <widget class="QTextBrowser" name="PIDBrowser"/>
     </item>
     <item>
      <widget class="QTextBrowser" name="parentPIDBrowser"/>
     </item>
     <item>
      <widget class="QTextBrowser" name="privateUsageBrowser"/>
     </item>
     <item>
      <widget class="QTextBrowser" name="threadsBrowser"/>
     </item>
     <item>
      <widget class="QTextBrowser" name="priorityBrowser"/>
     </item>
     <item>
      <widget class="QTextBrowser" name="priorityClassBrowser"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="stopRealTimeButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>959</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop real-time</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
